test/e2e/framework/security/apparmor.go

```diff
@@ -37,6 +36,12 @@ const (
 
 	loaderLabelKey   = "name"
 	loaderLabelValue = "e2e-apparmor-loader"
+
+	// Unconfined profile
+	profileNameUnconfined = "unconfined"
+
+	// The prefix to an annotation key specifying a container profile.
+	containerAnnotationKeyPrefix = "container.apparmor.security.beta.kubernetes.io/"
```

Comment:
Add a TODO comment to import this from the `k8s.io/api` once it's moved there

Reply:
ref https://github.com/kubernetes/kubernetes/pull/89225

Reply:
> Add a TODO comment to import this from the `k8s.io/api` once it's moved there

sure Thanks you review

Reply:
> ref #89225
updated.
@andrewsykim    I will follow this pr 89225